docs/source/quickstart.rst

```diff
@@ -30,71 +30,102 @@ For example, to run the command on sample data included in ASPIRE (a single ``sa
-#############
+2. Particle Extraction
+######################
 
-The ``simulation`` command simulates a virtual particle made up of multiple gaussian blobs, generates of set of (noisy) images,
-runs the ASPIRE pipeline to determine the estimated mean volume and estimated covariance on the mean volume,
-and runs evaluations on these estimated quantities (against the `true` values which we know from the simulation).
+Given a dataset of full micrographs (``*.mrc`` file) and corresponding coordinate files containing the locations
+of picked particles in the ``*.mrc``, the ``extract-particles`` command extracts the particles into one or more ``.mrcs``
+stacks and generates a ``.star`` file.
+
+Example usage:
+
+.. code-block::
+
+    aspire extract-particles --mrc_paths=my/data/sample.mrc --coord_paths=my/data/coords/sample.coord --starfile_out=my_dataset_stack.star --particle_size=256 --centers
+
+3. Estimate Contrast Transfer Function
+######################################
+
+The ``estimate-ctf`` command estimates the CTF from experimental data and returns the CTF as a mrc file.  For example,
 
 .. code-block:: console
 
-    aspire simulation
+      python -m aspire estimate-ctf --data_folder path_to_input_data_folder
 
-Use the ``--help`` argument to look for configurable options. You can select the no. of distinct gaussian blobs, the no. of images,
-the resolution of the (square) images generated etc.
+.. note::
 
-3. Reconstructing a mean volume with covariance
-###############################################
+    This command expects data files are in the directory prescribed by ``--data_folder``,
+    and will process all files with the extension ``.mrc`` and ``.mrcs`` contained there.
+    This command will output mrc files to a ``--output_dir``, set to ``./results`` by default.
 
-The ``cov3d`` command takes in a ``*.star`` file, processes the images (``*.mrcs`` files) found in the starfile, and runs the ASPIRE pipeline
-to determine the estimated mean volume and estimated covariance on the mean volume. No results are saved currently, but this command is
-a good way to exercise most parts of the ASPIRE pipeline.
+4. Image Preprocessing
+######################
 
-For example, to run the command on a sample data included in ASPIRE:
+The ``preprocess`` command takes in a ``*.star`` file containing raw images and applies a selection of preprocessing
+methods such as phase flipping, downsampling, normalization to background noise, noise whitening, and contrast invesrion.
+Resulting images are saved as a starfile.
+
+For example, to run the command on sample data included in ASPIRE:
 
 .. code-block:: console
 
-    aspire cov3d --starfile /path/to/aspire/data/sample_relion_data.star --pixel_size 1.338 --max_resolution 8 --cg_tol 0.2
+   aspire preprocess --starfile_in path/to/aspire/data/sample_relion_data.star --starfile_out preprocess_output.star --downsample 8
 
-.. note::
+Use the ``--help`` argument to look for configurable options.
```

Comment:
I saw this isn't added to all the commands. Maybe it should just go at the top or bottom since it applies to all of them? Or just add for each section

Reply:
Removed these comments from the individual sections and added to the top.

Reply:
You missed the one on line 68 😛

Reply:
Line 74 now

Reply:
AAH! oops. thanks! gone.